README.md

```diff
@@ -95,8 +95,14 @@ orchestrator.registerScenario('proper zome call', async (s, t) => {
 
   // and install a single happ
   const carol_blog_happ = await carol.installHapp([dnaBlog])
-  // or a happ with a previously generated key
-  const carol_test_happ_with_bobs_test_key = await carol.installHapp([dnaTest], bob_blog_happ.agent)
+
+  // or a happ with a previously generated key and previously registered dna by hash
+  // which you can get from the cells as returned in the InstalledHapp from previous install
+  // or from a call to registerDna which might look like this to create a derived dna using
+  // a different uuid
+  const blogDnaHash = carol_test_happ.cells[0].dnaHash()
+  const derivedDnaHash = await carol.registerDna({hash: blogDnaHash}, "1234567890")
```

Comment:
Should this say `testDnaHash`?

Reply:
In the example carol has a blogDna installed.  She then uses creates a derived DNA from that hash and registers a new derived dna.